src/Pages/Auth/Login/index.js

```diff
@@ -21,9 +25,39 @@ function Login(props) {
     history.push("/login")
   }
 
+  const [formData, updateFormData] = useState({ email: "", password: "" })
+
+  const handleChange = (e) => {
+    updateFormData({
+      ...formData,
+
+      // Trimming any whitespace
```

Comment:
I don't agree on trimming in this place. You can trim if you like in the handleSubmit function